deploy-deps.sh

```diff
@@ -37,13 +37,36 @@ check_req(){
 }
 
 deploy() {
+    test_pvc_binding
```

Comment:
I think this should be called from within `check_req` or right after it in main.